common/setups/rasr/util/rasr.py

```diff
@@ -0,0 +1,204 @@
+__all__ = [
+    "RasrDataInput",
+    "RasrInitArgs",
+    "RescoreArgs",
+    "RasrSteps",
+]
+
+
+from collections import OrderedDict
+from typing import Dict, Optional
+
+import i6_core.meta as meta
+
+
+class RasrDataInput:
+    """
+    this class holds the data information for a rasr gmm setup:
+    - corpus object
+    - lexicon: file, normalization
+    - lm: file, scale, type
+    - concurrency
+    """
+
+    def __init__(
+        self,
+        corpus_object: meta.CorpusObject,
+        lexicon: dict,
+        lm: Optional[dict] = None,
+        concurrent: int = 10,
```

Comment:
Is 10 a reasonable default?

also
```suggestion
        concurrent: Optional[int] = 10,
```
right?

Reply:
> Is 10 a reasonable default?

for a eval set, I think so. for training set maybe not..

> right?

No, I don't think so. concurrent variable needs to be set to an int. It is used directly in for example `SegmentCorpusJob` which does not accept a `None` value.

Reply:
> for a eval set, I think so. 

ok.

> No, I don't think so. [...] which does not accept a None value

yes, it was a misunderstanding on my side about the meaning of `Optional`. pls. disregard comment. :)